source/catch/main.cc

```diff
@@ -38,5 +40,5 @@
 #include "../test/integration_test/endosymbiosis/res_distribute.test.cc"
 #include "../test/integration_test/dirty_transmission/hz_mut_rate.test.cc"
 
-//#include "../PGGendtoend.test.cc"
-//#include "../test/end_to_end.test.cc"
+// #include "../PGGendtoend.test.cc"
```

Comment:
Please remove these since we're trying to keep them out finally

Reply:
Pretty sure they should be gone now